File: integrations/aws-strands/error-code-corpus/typescript/message_concatenation.ts
/** A message built by concatenating literals around an interpolated value. */
import { EventType, type BaseEvent } from "@ag-ui/core";

export function emit(detail: string): BaseEvent {
  return {
    type: EventType.RUN_ERROR,
    message: "Concatenated " + detail + " tail.",
    code: "MESSAGE_CONCATENATION",
  };
}

File: integrations/aws-strands/error-code-corpus/python/message_concatenation.py
"""A message built by concatenating literals around an interpolated value."""

from ag_ui.core import EventType, RunErrorEvent


def emit(detail: str) -> RunErrorEvent:
    return RunErrorEvent(
        type=EventType.RUN_ERROR,
        message="Concatenated " + detail + " tail.",
        code="MESSAGE_CONCATENATION",
    )
